tests/integration/backward_compatible/proxy/topic_test.py

```diff
@@ -44,3 +45,25 @@ def assert_event():
 
     def test_str(self):
         self.assertTrue(str(self.topic).startswith("Topic"))
+
+    def test_publish_all(self):
+        skip_if_client_version_older_than(self, "5.2")
+        collector = event_collector()
+        self.topic.add_listener(on_message=collector)
+
+        messages = ["message1", "message2", "message3"]
+        self.topic.publish_all(messages)
+
+        def assert_event():
+            self.assertEqual(len(collector.events), 3)
+
+        self.assertTrueEventually(assert_event, 5)
+
+    def test_publish_all_none_argument(self):
+        with self.assertRaises(AssertionError):
```

Comment:
you need to add this `skip_if_client_version_older_than(self, "5.2")` call to here as well. You need to guard all usages of new APIs in all tests if they are in the `backward_compatible` folder with such methods

Reply:
Now all tests are guarded, see https://github.com/hazelcast/hazelcast-python-client/pull/570/commits/b91a6e5018e63f4a14938e3680d9762d4586863e.